tests/core/featurizers/test_single_state_featurizers.py

```diff
@@ -225,6 +225,70 @@ def test_single_state_featurizer_with_entity_roles_and_groups(
     )
 
 
+@pytest.mark.timeout(300)  # these can take a longer time than the default timeout
+def test_single_state_featurizer_with_bilou_entity_roles_and_groups(
+    unpacked_trained_moodbot_path: Text,
+):
+    from rasa.core.agent import Agent
+
+    interpreter = Agent.load(unpacked_trained_moodbot_path).interpreter
```

Comment:
do we really need a fully trained model here?

Reply:
I believe there is no other way currently to get featurizers?

Reply:
at least other tests get the full model. I just copy pasted 🙈

Reply:
It's not ideal because it means we have to train moodbot just to run this test, and I'm assuming the functionality tested here doesn't require a trained model right?

Reply:
it does. Remember we discussed the need for separate featurizers? This is the reason, in order to extract entities we need trained tokenizer, and I don't think there is a way extract only tokenizer. And if there is, I'd say it is out of scope of current PR

Reply:
ok makes sense